### ISSUE TITLE
Suggestion: option for minimum loading time

### ISSUE THREAD

**Issue by sheriffderek** [mike-north/ember-load#32]
With these fancy SPA type apps - things happen faster, and that's great - but they also seem to feel clunky in many scenarios because of the action timing doesn't follow any underlying "beat." (which there never was before, but we were used to the full page load)

In this case, the app may load so fast that the loading spinner appears like a broken flash instead of a loading indicator. In many cases, I would rather my loading states complete a minimum interval of visibility before moving on / even if the content is already ready. Maybe something in the config like this:

```
module.exports = function(environment) {
  var ENV = {
    // optional
    'ember-load': {
      // This is the default value, if you don't set this opton
      loadingIndicatorClass: 'ember-load-indicator',
      min-interval: 1200
    }
    ...
  }
  ...
}  
```

Any thoughts on that? I've been working with https://github.com/GavinJoyce/ember-delay for route loading and it feels nice.  Example:  http://august.services

**Comment by mike-north** [comment]
I think I'd rather leave this outside the addon. What you're asking for could be accomplished easily using [Ember.Application.deferReadiness](http://emberjs.com/api/classes/Ember.Application.html#method_deferReadiness), and I'd rather let users do their own thing using the built-in framework mechanism

**Comment by sheriffderek** [comment]
OK. I'll check that out.Thanks!

**Comment by brendanoh** [comment]
I used this with defer readiness to get a delay but it still feels incomplete to me. While this does allow the loading to take longer it still seems to be a solution for only the simplest of css loaders. The ability to fade out or cross fade into the app would make this feel like a complete solution

Previously I had used ember-cli-preloader and I was hoping this would be a good replacement.

Not trying to be critical Mike as I love your stuff but just my thoughts on a closed thread.
